tests/Run-Tests.ps1

```diff
@@ -186,7 +190,7 @@ function ExecTest($argList)
 if ($TestDirectory -eq "")
 {
     $MyPath = (get-item $MyInvocation.MyCommand.Path).Directory.FullName;
-    $TestSrcDirectory = Join-Path $MyPath "..\build\$Platform\$Config\Tests"
+    $TestSrcDirectory = Join-Path $MyPath "..\build\$Platform\$Config\Universal Windows\"
```

Comment:
Let me know when you get a daily build out with published bits and then we can see what changes need to be made.


---
In reply to: [102321885](https://github.com/Microsoft/WinObjC/pull/1971#discussion_r102321885) [](ancestors = 102321885,102313367,102312399)